addons/udes_stock/tests/plot_load_tests.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from scipy.stats import spearmanr
+from collections import defaultdict
+from matplotlib import pyplot as plt
+import sys
+
+COEFFICIENT_THRESHOLD = 0.7
+PVAL_THRESHOLD = 0.001
+
+# Do we want to bootstrap this?
```

Comment:
bootstrap?

Reply:
Randomly selecting values from the list of times instead of taking the means then repeating the process.  This could be used to account for there being a range of values but it would obviously be slower. Probably not worth it.